### ISSUE TITLE
Allow axe.source to run in strict mode

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#349]
When running webdriver selenium in Firefox, `executeScript()` will run in script mode, thus `this` no longer refers to the window object. Because of that, the following won't work:

```js
axe.source = '(' + axeFunction.toString() + ')(this, this.document);';
```

**Comment by dylanb** [comment]
@WilcoFiers What version of Firefox does this break in?

**Comment by marcysutton** [comment]
Fixed in #355.
